alternator/executor.cc

```diff
@@ -883,7 +895,13 @@ future<executor::request_return_type> executor::list_tags_of_resource(client_sta
     return make_ready_future<executor::request_return_type>(make_jsonable(std::move(ret)));
 }
 
-static void verify_billing_mode(const rjson::value& request) {
+struct billing_mode_type {
+    bool provisioned = false;
+    std::string rcu;
+    std::string wcu;
```

Comment:
Why is this a string?

Reply:
so we can save it in a tag; the alternative is to do the int-to-string conversion when storing the value.